zhaquirks/tuya/ts001x.py

```diff
@@ -212,8 +356,8 @@ class TuyaDoubleNoNeutralSwitch_2(EnchantedDevice, TuyaSwitch):
     }
 
 
-class TuyaTripleNoNeutralSwitch(EnchantedDevice, TuyaSwitch):
-    """Tuya 3 gang no neutral light switch."""
+class Tuya_TripleNoNeutralSwitch_2(EnchantedDevice, TuyaSwitch):
```

Comment:
Maybe remove the underscore between ``Tuya`` and ``TripleNoNeutralSwitch`` here, as the others also don't have one. (The one for ``_2`` is fine)

Also, should ``TuyaTripleNoNeutralSwitch_2`` be moved under ``TuyaTripleNoNeutralSwitch``, as they're really similar? (So "_2" quirks directly under the "main quirk" of each device?)

Reply:
https://github.com/MattWestb/zha-device-handlers/blob/1fb77ea83c7dfe32cd6beeb3bbcb814eaecaf7b2/zhaquirks/tuya/ts001x.py#L142
is over 
https://github.com/MattWestb/zha-device-handlers/blob/1fb77ea83c7dfe32cd6beeb3bbcb814eaecaf7b2/zhaquirks/tuya/ts001x.py#L359

The rest is one large mess with failing parts of signature and so on so i not touching it for the moment.

Normally new families / version is coming on all 3 types and its good having  them together as was done from the beginning.